ci/mypy-stubtest-allowlist.txt

```diff
@@ -1,7 +1,7 @@
 # Non-typed (and private) modules/functions
 matplotlib\.backends\..*
 matplotlib\.tests(\..*)?
-matplotlib\.pylab\..*
+matplotlib\.pylab(\..*)?
```

Comment:
Without this change, stubtest both complains that it can't find a stub for pylab and that the allowlist entry is unused.  Removing the entry only solved the second complaint.  It seemed intuitive to make this line look like the `sphinxext` line.